go/vt/vtgate/planbuilder/operators/sharded_routing.go

```diff
@@ -234,6 +237,24 @@ func (tr *ShardedRouting) searchForNewVindexes(ctx *plancontext.PlanningContext,
 	return nil, newVindexFound
 }
 
+func (tr *ShardedRouting) planBetweenOp(ctx *plancontext.PlanningContext, node *sqlparser.BetweenExpr) (routing Routing, foundNew bool) {
+	// x BETWEEN a AND b => x >= a AND x <= b
+	column, ok := node.Left.(*sqlparser.ColName)
+	if !ok {
+		return nil, false
+	}
+	vals := []sqlparser.Expr{}
+	vals = append(vals, node.From, node.To)
+	var vdValue sqlparser.ValTuple = vals
```

Comment:
can be simplified to
`vdValue := sqlparser.ValTuple([]sqlparser.Expr{node.From, node.To})`

Reply:
thanks for snippet, incorporated in  40d041f28f5b371854b4538ccc915b0a32f0b79d